libsolidity/experimental/codegen/IRGeneratorForStatements.cpp

```diff
@@ -201,13 +201,50 @@ void IRGeneratorForStatements::endVisit(BinaryOperation const& _binaryOperation)
 	Type functionType = helper.functionType(helper.tupleType({leftType, rightType}), resultType);
 	auto [typeClass, memberName] = m_context.analysis.annotation<TypeInference>().operators.at(_binaryOperation.getOperator());
 	auto const& functionDefinition = resolveTypeClassFunction(typeClass, memberName, functionType);
-	// TODO: deduplicate with FunctionCall
+	std::string functionDeclaration = var(_binaryOperation).commaSeparatedList();
```

Comment:
```suggestion
	std::string result = var(_binaryOperation).commaSeparatedList();
```

In what way is this a "functionDeclaration" :-)? It appears to be the name of the variable(s) storing the result of the binary operation.

Reply:
Ops, yes. Fixed.